modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/GetEntryRenderDataMVCResourceCommand.java

```diff
@@ -641,7 +641,9 @@ else if (ctEntry.getChangeType() !=
 			JSONArray workflowActionsJSONArray = _getWorkflowActionsJSONArray(
 				ctEntry, rightModel, themeDisplay, resourceResponse);
 
-			if (workflowActionsJSONArray != null) {
+			if ((workflowActionsJSONArray != null) &&
+				(ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT)) {
```

Comment:
We should apply this check before getting the `workflowActionsJSONArray` so that we don't waste a call to retrieve the data.